### DISCUSSION TITLE
Max players and setting actual amount of players

### DISCUSSION THREAD

**Discussion by worldpeace-germany** [missionpinball/missionpinball#78] [Q&A] [answered]
Hello,

let assume I want to allow at max 27 players. Today only 8 players are going to play. How do I configure the maximum and how to tell the system upon start that only 8 players are present? I thought pressing 8 times the start button, but that doesn't seem to have an effect. (using the tutorial code step 15).

Thanks!

**Comment by cobra18t** [comment]
Do you have a config example of what you tried? 

That is essentially what I have done, but my max_players is a service mode setting between 1 and 4.

**Comment by worldpeace-germany** [comment]
Hello,

yes from the tutorial step 15, https://github.com/missionpinball/mpf-examples/blob/dev/tutorial/step_15/config/config.yaml  The main problem might be that I have not read anything (because didn't find anything) how to configure multiple users and how to select how many players are actually play. At the moment I am simply guessing how it might work.

In that context I tried to display somehow the settings mpf is using, https://docs.missionpinball.org/game_vars/index.html

But I am not able to display these variables, see https://github.com/orgs/missionpinball/discussions/77

Any ideas?

Thanks!

**Reply by cobra18t** [reply]
To start a multiplayer game, the max_players needs to be >1 (the default is 4) and you press start multiple times. One event that is triggered when another player is added is player_added: https://docs.missionpinball.org/events/player_added.html?highlight=player_added

There may be some helpful info in this guide: https://docs.missionpinball.org/displays/slides/multiplayer_display.html

**Reply by cobra18t** [reply, marked as answer]
Also, here is an example CobraPin config that explicitly sets the max_players to 4. You could change that config line to 27.
https://docs.missionpinball.org/hardware/opp/cobrapin/index.html#example-config

**Comment by worldpeace-germany** [comment]
thanks, problem solved.
